app/Models/Post.php

```diff
@@ -177,6 +177,44 @@ public function toBlinkPayload()
         ];
     }
 
+    /**
+     * Transform the post model for Quasar.
+     *
+     * @return array
+     */
+    public function toQuasarPayload()
+    {
+        return [
+            'id' => $this->id,
+            'signup_id' => $this->signup_id,
+            'northstar_id' => $this->northstar_id,
+            'type' => $this->type,
+            'action' => $this->action,
+            'quantity' => $this->quantity,
+            // Add cache-busting query string to urls to make sure we get the
+            // most recent version of the image.
+            // @NOTE - Remove if we get rid of rotation.
+            'media' => [
+                'url' => $this->getMediaUrl(),
+                'original_image_url' => $this->url . '?time='. Carbon::now()->timestamp,
+                'caption' => $this->caption,
+            ],
+            'tags' => $this->tagSlugs(),
+            'reactions' => [
+                'total' => isset($this->reactions_count) ? $this->reactions_count : null,
+            ],
+            'status' => $this->status,
+            'source' => $this->source,
+            'remote_addr' => $this->remote_addr,
+            'details' => $this->details,
+            'created_at' => $this->created_at->toIso8601String(),
+            'updated_at' => $this->updated_at->toIso8601String(),
```

Comment:
would there ever be a need for us to send `deleted_at`? Not sure how they count signups/posts that have been deleted?

Reply:
@chloealee ooooo great question! currently I think deleting a post would update the timestamp on the signup and then Quasar re-ingests that and sees that the deleted post is no longer there. I guess I need to make sure that deleting a post goes through the PostService?

Reply:
@katiecrane oh that's right!

Reply:
Oh seeing this now! 🆙

I'm a little concerned we're "leaking" implementation details by sending along the `deleted_at` column rather than a distinct "deleted" event. Let's double-check with Team Storm to make sure we're all on the same page.

Reply:
Whatever works best for Storm is cool with me!